calc/division.py

```diff
@@ -0,0 +1,3 @@
+
+def divide(x,y):
+    return int(x/y)
```

Comment:
Please don't cast to `int`. Also, add test for case when dividing result is not int. For example:  9/2 = 4.5